src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -153,32 +149,22 @@ public static unsafe int IndexOf(ref byte searchSpace, byte value, int length)
                         index += Vector<byte>.Count;
                         continue;
                     }
-                    // Found match, reuse Vector vComparison to keep register pressure low
```

Comment:
That's [correct](https://github.com/dotnet/corefx/pull/23510/files/e5df9926944323f1b8a9b8b1d9884dbf73aa2a45#diff-b3f745bb78e27559dcdeacdc3bcdab84)